buildSrc/src/main/rocker/com/uber/okbuck/template/config/OkbuckTargets.rocker.raw

```diff
@@ -242,6 +243,7 @@ def okbuck_manifest(
         out = 'AndroidManifest.xml',
         type = 'okbuck_manifest',
         cmd = ' '.join(cmds),
+        visibility = visibility,
```

Comment:
this rule was not previously public and now it is. Maybe it is better to keep it private?